### ISSUE TITLE
Integrate `dotnet run` with `wasmtime` args

### ISSUE THREAD

**Issue by richlander** [dotnet/sdk#32621]
I'm using `--mapdir`. That means I cannot use `dotnet run`, which leaves me very sad. Perhaps we need a `host-args.txt` or something similar.

My current workflow: https://github.com/richlander/wasm-samples/blob/main/tomarkup/README.md

**Comment by richlander** [comment]
Yes. I'd suggest we close this one as a dupe, however, it seems better to have SDK oriented issues open on dotnet/sdk not dotnet/runtime.

**Comment by pavelsavara** [comment]
I think the implementation is there https://github.com/dotnet/runtime/tree/main/src/mono/wasm/host
